### DISCUSSION TITLE
INNOCN 27C1U Adjusting the sound of the monitor does not work

### DISCUSSION THREAD

**Discussion by caifenglin** [MonitorControl/MonitorControl#820] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

On my joint innovation display, the sound does not work, and the brightness can be successfully adjusted to display joint innovation 27c1u，ddcctl is used to adjust the sound without error, but it has no effect

### Expected behavior

I hope it can support adjusting the sound of the monitor，Display parameter link https://item.jd.com/100013814761.html，I hope the sound adjustment function can be repaired. Thank you very much

### Environment Information (please complete the following information)

```markdown
- macOS version:10.15.7
- Mac model:
- MonitorControl version:4.0.2
- Monitor(s):27C1U
- Monitor Cable(s)/Connection(s): Display Port
- Apple Silicon/M1 (yes or no):no
```

**Comment by waydabber** [comment]
Hi @caifenglin - I am afraid this is a display specific issue. I advise you to try changing the volume via alternate apps that use DDC for control, like [ddcctl](https://github.com/kfix/ddcctl) or [Lunar](https://lunar.fyi) (@alin23). If neither of the works then the display is probably incompatible with the DDC command that adjusts the volume and there is nothing we can do about it I am afraid. :(

I'll convert this into a discussion so that others with similar displays might join and share their experience.

**Comment by heliumllc** [comment]
Any volume fixes for INNOCN monitors?

**Comment by ToSupply** [comment]
Hi,
Same here, the brightness can be adjusted but not the sound with this display:
EIZO FlexScan EV3285 BK schwarz / 31,5" / 16:9 4K UHD

Is there something that should be configured in MacOS in order for it to work?
Thanks for your help on this,
Best

**Comment by patcbuilds** [comment]
Just adding here that I recently got a 28C1Q (the vertical 2560x2880 screen) and I was able to get mute working but not volume in BetterDisplay by changing a setting. BetterDisplay has an option to switch the mute/unmute values.

<img width="583" alt="image" src="https://github.com/MonitorControl/MonitorControl/assets/3843748/3a349685-a8ba-4976-8066-3678d5cdc1b9">

Does anyone have a Windows laptop where they can run the ControlMyMonitor utility and see all of the available DDC hex command codes? I'm guessing we just need to find the right DDC control code for volume and it's not the default hex value `62`.

**Reply by Junnanbunan** [reply]
![IMG_0553](https://github.com/user-attachments/assets/29ad2587-d21d-459d-b1ba-20afad453b1a)
There isn't volume vcp appeared in ControlMyMonitor
